src/nevermined/api/SearchApi.ts

```diff
@@ -193,4 +208,238 @@ export class SearchApi extends Instantiable {
     }
     return this.query(query)
   }
+
+  /**
+   * Search of all subscriptions created by `account`
+   *
+   * @param account - The account that created the subscriptions.
+   * @param offset - The number of results to return
+   * @param page
+   * @param sort - The sort order
+   * @param appId - The appId used to filter the results
+   *
+   * @returns {@link Promise<QueryResult>}
+   */
+  public async subscriptionsCreated(
```

Comment:
this not only will get the subscriptions also the services, filter by `nft-sales` services is need in order to get just subcriptions

Reply:
only subscription DDOs have `nftType === 'nft721-subscription'` that is enough to get subscriptions

service DDOs have `nftType === 'nft721'`